### PR TITLE
Reviewer RKD: Create GR schema

### PR DESCRIPTION
**Pull request by graemerobertson**

### CODE REVIEW

File: homer.root/usr/share/clearwater/cassandra-schemas/homer.sh [hunk 1]
```diff
@@ -16,7 +17,24 @@ then
     /usr/share/clearwater/bin/poll_cassandra.sh --no-grace-period
   done
 
-  echo "CREATE KEYSPACE homer WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 2};
+  replication_str="{'class': 'SimpleStrategy', 'replication_factor': 2}"
+
+  # If local_site_name and remote_site_names are set then this is a GR
```
Comment (rkd-msw) [inline homer.root/usr/share/clearwater/cassandra-schemas/homer.sh]: Should this be in a common Bash file that homestead_provisioning.sh, homer_provisioning.sh etc. just dot in? Makes it easier to change this in future.

File: homer.root/usr/share/clearwater/cassandra-schemas/homer.sh [hunk 2]
```diff
@@ -16,7 +17,24 @@ then
...
+    replication_str="{'class': 'NetworkTopologyStrategy', '$local_site_name': 2"
+    for remote_site in "${remote_site_names_array[@]}"
+    do
+      # Set the replication factor for each site to 2.
+      replication_str+=", '$remote_site': 2"
+    done
+    replication_str+="}"
+  fi
+
+  echo "CREATE KEYSPACE homer WITH REPLICATION = $replication_str;
```
Comment (rkd-msw) [inline homer.root/usr/share/clearwater/cassandra-schemas/homer.sh]: Not your code, but is $namespace_prefix set up on line 39?

**Comment by rkd-msw** [comment]
For the record, we discussed upgrade, and this just automates a previously manual step - so all existing and all new GR deployments will be consistent.

File: homer.root/usr/share/clearwater/cassandra-schemas/homer.sh (hunk 1 above)
Comment (graemerobertson) [inline homer.root/usr/share/clearwater/cassandra-schemas/homer.sh]: Good idea - I'll move it to the clearwater-cassandra repo (but still install the Bash file to /usr/share/clearwater/cassandra-schemas)

**Pull request merged**